GiniVision/Classes/Core/Screens/Document picker/Gallery/ImagePickerViewController.swift

```diff
@@ -67,11 +67,7 @@ final class ImagePickerViewController: UIViewController {
         
         title = currentAlbum.title
         
-        if #available(iOS 13.0, *) {
-            collectionView.backgroundColor = .systemBackground
-        } else {
-            view.backgroundColor = .white
-        }
+        view.backgroundColor = UIColor().colorFromGiniColor(giniColor: giniConfiguration.galeryScreenBackgroundColor)
```

Comment:
By renaming and making the method static we can call it like this:
```suggestion
        view.backgroundColor = UIColor.from(giniColor: giniConfiguration.galeryScreenBackgroundColor)
```